### PR TITLE
docs(fall-menu):[fall-menu]text correction

### PR DESCRIPTION
**Pull request by James-9696**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: examples/sites/demos/pc/app/fall-menu/custom-menuitem-composition-api.vue [hunk 1]
```diff
@@ -2,13 +2,15 @@
   <div class="left-panel">
     <tiny-fall-menu :data="dataset">
       <template #level1="data">
-        <a>{{ data.slotScope.title + 'level1' }}</a>
+        <a>{{ data.slotScope.title + ' ' + 'level1' + ' ' }}</a>
```
Comment (zzcr) [inline examples/sites/demos/pc/app/fall-menu/custom-menuitem-composition-api.vue]: 这里用模板字符串要好点

File: examples/sites/demos/pc/app/fall-menu/webdoc/fall-menu.js [hunk 2]
```diff
@@ -44,10 +44,10 @@ export default {
     {
       'name': 'fall-menu',
       'type': 'component',
-      'properties': [
+      'props': [
         {
           'name': 'data',
-          'type': 'object , Array',
+          'type': 'Array',
```
Comment (zzcr) [inline examples/sites/demos/pc/app/fall-menu/webdoc/fall-menu.js]: 这写成object[] 要使用ts的类型去标注属性的prop

File: examples/sites/demos/pc/app/fall-menu/custom-menuitem-composition-api.vue (hunk 1 above)
Comment (James-9696) [reply examples/sites/demos/pc/app/fall-menu/custom-menuitem-composition-api.vue]: 已修改

File: examples/sites/demos/pc/app/fall-menu/webdoc/fall-menu.js (hunk 2 above)
Comment (James-9696) [reply examples/sites/demos/pc/app/fall-menu/webdoc/fall-menu.js]: 已修改

**Review by zzcr — APPROVED** [review]

**Comment by kagol** [comment]
@all-contributors please add @James-9696 for doc

**Pull request merged**